packages/shared/src/environment/index.ts

```diff
@@ -0,0 +1,19 @@
+import { parseEnv } from 'znv'
+import { z } from 'zod'
+
+//INFO
+// As a convention all feature flags should be prefixed with a FF_
+const featureFlagSchema = z.object({
+  FF_AUTOMATE_MODULE_ENABLED: z.boolean().default(true) // TODO: Set to default and figure out deployment settings
```

Comment:
@gjedlicska  @iainsproat  For now I've set this to default to true in feature flags, to avoid having to deal with the hassle of setting up this value in all envs, but when you plan to merge the PR in you should undo this